cypress.json

```diff
@@ -3,6 +3,9 @@
     "defaultCommandTimeout": 10000,
     "video": false,
     "env": {
-        "youth_pass_url": "https://mbta.preprod.simpligov.com/preprod/portal/ShowWorkFlow/AnonymousEmbed/86cadfa6-e8ea-46f1-b6e8-62498f066962"
+        "youth_pass_url": "https://mbta.preprod.simpligov.com/preprod/portal/ShowWorkFlow/AnonymousEmbed/86cadfa6-e8ea-46f1-b6e8-62498f066962",
+        "youth_pass_dashboard_url": "https://mbta.preprod.simpligov.com/preprod/portal/Dashboard",
+        "malden_test_username": "maldentestuser@mbta.com",
```

Comment:
Is there a better way to manage these creds?

Reply:
This is fine. Since they are just hard coded anyway and not overridden, I can also just see defining them at the top of the test file too as another option.